docs/User/ProjectRelated/Changelog.md

```diff
@@ -3,6 +3,38 @@
 <!-- See rendered changelog at https://streamly.composewell.com -->
 ## 0.9.0 (Unreleased)
 
+### Breaking changes
+
+Multiple combinators now only work with `SerialT`. All other types need to be
+adapted accordingly using `adapt`. Any combinator `f` should now become `f
+. adapt`.
+
+Combinators that work with arrays require an `Unbox` constraint instead of
+`Storable`.
+
+Imports that may be required:
+```
+import Streamly.Prelude (adapt)
+import Streamly.Data.Array.Unboxed (Unbox)
+```
+
+* `Streamly.Unicode.Stream`
+  * Encoding and decoding combinators need to be adapted accordingly.
+
+* `Streamly.Network.Socket`
+  * `writeChunks` and `writeChunk` need an `Unbox` constraint instead of
+    `Storable`.
```

Comment:
Instead of making these separate bullet points, these can just be mentioned under the relevant points above. We should try to keep the changelog as brief as possible.